homeassistant/components/ialarmxr/alarm_control_panel.py

```diff
@@ -0,0 +1,73 @@
+"""Interfaces with iAlarmXR control panels."""
+
+from homeassistant.components.alarm_control_panel import AlarmControlPanelEntity
+from homeassistant.components.alarm_control_panel.const import (
+    SUPPORT_ALARM_ARM_AWAY,
+    SUPPORT_ALARM_ARM_HOME,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import IAlarmXRDataUpdateCoordinator
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up a iAlarmXR alarm control panel based on a config entry."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities([IAlarmXRPanel(coordinator)])
+
+
+class IAlarmXRPanel(CoordinatorEntity, AlarmControlPanelEntity):
+    """Representation of an iAlarmXR device."""
+
+    def __init__(self, coordinator: IAlarmXRDataUpdateCoordinator) -> None:
+        """Initialize the alarm panel."""
+        super().__init__(coordinator)
+        self.coordinator: IAlarmXRDataUpdateCoordinator = coordinator
+
```

Comment:
```suggestion
        self._attr_unique_id = coordinator.mac
        self._attr_device_info = DeviceInfo(
            identifiers={(DOMAIN, self.unique_id)},
            manufacturer="Antifurto365 - Meian",
            name=self.name,
        )
```

Reply:
(I think you should make these changes yourselves, since it was hard for me to get the indent right via GitHub...)